jira/client.py

```diff
@@ -121,6 +121,18 @@ def _get_template_list(data):
     return template_list
 
 
+def _field_worker(fields=None, **fieldargs):
+    data = {}
+    if fields is not None:
+        data['fields'] = fields
+    else:
+        fields_dict = {}
+        for field in fieldargs:
+            fields_dict[field] = fieldargs[field]
+        data['fields'] = fields_dict
```

Comment:
Isn't this just equivalent to `data['fields'] = fieldargs.clone()`? Which itself is probably equivalent to `data['fields'] = fieldargs` (unless the fields dict is being mutated, which I'd consider very unexpected.)

What am I missing...?